documentation/docs/fundamentals/defining-your-theme.md

```diff
@@ -34,6 +34,16 @@ const theme = createTheme({
     l: 24,
     xl: 40,
   },
+  textVariants: {
+    header: {
```

Comment:
Can we also add in a default as it's also required. We can leave it as an empty object to show that it doesn't need to have a defined style.

```
defaults: {
// We can define a default text variant here.
},
```

Reply:
Good idea, fixed!